tests/test_route.py

```diff
@@ -0,0 +1,56 @@
+from urllib.request import urlopen
+
+import pytest
+from app.create_app import create_app
+from flask import url_for
+
+
+@pytest.fixture(scope="session")
+def app():
+    app = create_app()
+    return app
+
+
+@pytest.mark.usefixtures("live_server")
+class TestSearchPage:
+    """
+    GIVEN class checks if route for send_notification
+    is up & running
```

Comment:
:wrench: Not sure what this docstring means, please make sure you understand the use of GIVEN-WHEN-THEN. It's a standard method of describing what checks should do, some programs go even further and actually allow you to write checks using this language (See [Behave](https://behave.readthedocs.io/en/stable/api.html) or the much maligned [cucumber ](https://cucumber.io/docs/gherkin/reference/) ).

Reply:
Updated 👍